progress-reports/infrastructure/220916-infrastructure-progress-report.md

```diff
@@ -0,0 +1,46 @@
+## Intro
+
+This is a progress update for infrastructure provided by sudoblock including development of the tracker, indexer, and operation of RPC endpoints being used to support the ICON ecosystem for the period from August 1st to September 17th.
+
+## Summary
```

Comment:
All of these bullet points belong in the "accomplished this cycle" section, along with a reference to their issue, commit, or release.